src/components/RadioButton/RadioButton.tsx

```diff
@@ -32,7 +32,7 @@ export const RadioButton: React.FC<RadioButtonProps> = ({
       <input
         tabIndex={-1}
         className='RadioButton__input'
-        type='radio'
+        type='button'
```

Comment:
there has to be only one `radio` role within this component and it's set on the root level of the component

Reply:
I'd rather make this `type='hidden'` in this case.